packages/shared/src/modules/friends/libs/enums/friend-request-validation-message.enum.ts

```diff
@@ -0,0 +1,5 @@
+const FriendRequestValidationMessage = {
```

Comment:
```suggestion
const FriendValidationMessage = {
```
do not forget about file-name
please, follow the style of the rest of the application